application/helpers/SurveyRuntimeHelper.php

```diff
@@ -1729,6 +1729,15 @@ private function setSurveySettings($surveyid, $args)
         extract($args);
 
         $this->aSurveyInfo                 = getSurveyInfo($this->iSurveyid, App()->getLanguage());
+        if (isset($args['popuppreview']) && $args['popuppreview']) {
+            $this->aSurveyInfo['showxquestions'] = 'N';
+            $this->aSurveyInfo['shownoanswer'] = 'N';
+            $this->aSurveyInfo['showwelcome'] = 'N';
```

Comment:
Why hide it ?

Reply:
See https://github.com/LimeSurvey/LimeSurvey/pull/3659#discussion_r1447718138

Reply:
Still don't understand why hide it : if user want to show it : 

- He want to have it when previewing
- He want to have it doing a print HTML page for reviewing.

Reply:
Because that was the intention. It was a decision made by the management, The user chooses a survey in a very specific flow and I was explicitly asked to hide the Welcome. If you dislike this idea, please discuss it with Jory.